src/guc.c

```diff
@@ -150,6 +151,17 @@ _guc_init(void)
 							    /* check_hook= */ ts_license_update_check,
 							    /* assign_hook= */ ts_license_on_assign,
 							    /* show_hook= */ NULL);
+
+	DefineCustomStringVariable( /* name= */ "timescaledb.last_tuned",
+							    /* short_dec= */ "last tune run",
```

Comment:
I know this is being consistent with the previous one, and a bit of bike-shedding, but i think having the comment after the value is more readable.